images/cluster-setup/install.sh

```diff
@@ -122,33 +122,28 @@ install_tektoncd() {
   printf "\n"
 }
 
-wait_for_resource() {
-  args=( "$@" )
-  while ! kubectl get "${args[@]}" >/dev/null 2>/dev/null; do
-    sleep 10
-  done
-}
-
-wait_for_pod() {
-  args=( "$@" )
-  export -f wait_for_resource
-  if ! timeout 20s bash -c "wait_for_resource pods ${args[*]}"; then
-      exit_error "Pod not found."
-  fi
-  podname="$(kubectl get pods "${args[@]}" -o jsonpath='{.items[0].metadata.name}')"
-  kubectl wait --for=condition=Ready "pod/$podname" -n openshift-pipelines --timeout=60s >/dev/null
-}
-
 postchecks() {
   printf "Checking deployment\n"
   #checking if the pipelines and triggers pods are up and running
   for i in "${!clusters[@]}"; do
     switch_cluster
     printf "  - %s:\n" "${clusters[$i]}"
-    for pod in tekton-pipelines-controller tekton-triggers-controller tekton-triggers-core-interceptors; do
-      printf "    - %s: " "$pod"
-      wait_for_pod -n openshift-pipelines -l=app="$pod"
-      printf "OK\n"
+    for deploy in tekton-pipelines-controller tekton-triggers-controller tekton-triggers-core-interceptors; do
+      printf "Checking the status of %s: " "$deploy"
+
+      #a loop to check if the deployment exists
+      if ! timeout 300s bash -c "while ! kubectl get deployment/$deploy -n openshift-pipelines >/dev/null 2>/dev/null; do printf '.'; sleep 10; done"; then
```

Comment:
`printf "......- Deployed: "`. Replace '.' with spaces, as spaces were not displayed properly.